sonarr/sonarr.go

```diff
@@ -398,3 +398,19 @@ func (s *Sonarr) GetHistory(maxRecords int) (*History, error) {
 
 	return &history, nil
 }
+
+// MarkHistoryItemAsFailed marks the given history item as failed by id.
+func (s *Sonarr) MarkHistoryItemAsFailed(id int64) error {
```

Comment:
Let's rename the method to `Fail()`. simple and ez.

What actually this ID? Is it a specific `historyID`? Is it a `seriesID`? Perhaps it's an `episodeID`? Can you rename `id` to be more descriptive?

Reply:
Renamed and I updated the `id` to be `historyID` 👍